test/helpers/node.go

```diff
@@ -106,6 +106,8 @@ func (s *SSHMeta) ExecWithSudo(cmd string) *CmdRes {
 
 // Exec returns the results of executing the provided cmd via SSH.
 func (s *SSHMeta) Exec(cmd string) *CmdRes {
+	log.Debugf("running command: %s", cmd)
```

Comment:
This is good, but if you run the test with `-v` you will get too much information no?

Reply:
In the tests I was looking at, it was the right amount of information and made it convenient for me to use `-cilium.holdEnvironment` and copy/paste the command to try it myself, observe the output, debug the live system. Knowing the exact command that failed is super useful for this kind of case.

I haven't looked closely at how it affects the output for other tests so I'd welcome everyone to share their thoughts here. We could even trial run this and if it seems too verbose, try something different. Personally I like to run local tests without `-v`, then if I hit some specific test failure, focus that test and turn up the verbosity.